### DISCUSSION TITLE
fallback: true and revalidate - Does the page update once the re-validation occurs?

### DISCUSSION THREAD

**Discussion by dotjosh** [vercel/next.js#14821] [Help]
In the below setup, I was under the impression that the following workflows would occur:

- User 1 hits page 
  - router.isFallback = true (can show loading message) 
  - User gets static page (with props.isFallback == false) and rendered props A

- User 2 hits page 15 seconds later (more than the revalidate setting)
  - Is sent User 1 static page with rendered props A
  - _New page is statically rendered in the background with props B_
  - Page is updated with props B **<--- this step never happens**

So even though I have revalidate set to 10 seconds.  The second user, who could show up 3 days later, might see 3-day-old data  unless they manually refresh the page.

Am I doing something wrong, or was this the intended behavior?

```javascript
export async function getStaticPaths() {
	return {
		paths: [],
		fallback: true,
	};
};

export async function getStaticProps({ params }){
	return {
		unstable_revalidate: 10,
		props: {
			date: moment().format(),
		},
	};
};
```

**Comment by timneutkens** [comment]
> Am I doing something wrong, or was this the intended behavior?

It's the intended behavior, similar to the `stale-while-revalidate` directive.

**Reply by dotjosh** [reply]
@timneutkens I suppose I am going to create an external cron job to loop thru the sites to trigger a rebuild for each expected fallback value so the pages never get too stale.    

Do you have any thoughts on that or a better way to deal with it?  In my use case, 60 seconds stale is OK, but 3 days is not.
